### PR TITLE
added return button

### LINKED ISSUE #45: Return from Roles
### Outcome

As a user, I can return to the rest of the website from the roles page

### Moment

The user is able to exit the roles page without closing the website or having to log back in. They can smoothly return to their profile page with their settings saved.

### Key Results

- [x] User can exit the roles page using a link on the page
- [x] User is returned to their profile page upon exit

### PR DESCRIPTION
**Pull request by luke-mccallum**
return button added with basic styling in line with the rest of the existing page

### CODE REVIEW

File: frontend/src/pages/Roles/Roles.tsx [hunk 1]
```diff
@@ -30,11 +37,15 @@ function Roles() {
                     <div className="h-[300px]">
                         <UserList allUsers={allUsers} setActiveRole={setActiveRole} />
                     </div>
+                    {/* Return button */}
+                    <div className="mt-10">
+                        <button className="text-3xl text-[#ffffff] bg-[#262b6c] p-4 border-4" onClick={returntoProfile} >Return to Profile</button>
```
Comment (grantliunz) [inline frontend/src/pages/Roles/Roles.tsx]: maybe add a "rounded-md" and change border to "border-2". Just for consistency across components

**Review by grantliunz — CHANGES_REQUESTED** [review]

File: frontend/src/pages/Roles/Roles.tsx (hunk 1 above)
Comment (luke-mccallum) [reply frontend/src/pages/Roles/Roles.tsx]: done

**Review by long715 — APPROVED** [review]
LGTM! Just a concern on the functionality since we will be adding the sidebar to the roles page as well. Will finalise this when the HIFI wireframe for the roles page comes out.

**Comment by luke-mccallum** [comment]
> LGTM! Just a concern on the functionality since we will be adding the sidebar to the roles page as well. Will finalise this when the HIFI wireframe for the roles page comes out.

yep no worries there seeing as it's in the same div as the rest of the section on the left so can be adjusted alongside it

**Review by grantliunz — APPROVED** [review]
lgtm👍

**Pull request merged**